src/engine/routing_algorithms/map_matching.cpp

```diff
@@ -419,42 +420,32 @@ mapMatchingImpl(SearchEngineData &engine_working_data,
     return sub_matchings;
 }
 
-SubMatchingList
+template SubMatchingList
```

Comment:
Here is "14.7.2 Explicit instantiation" with syntax `extern_opt template declaration` and not a template specialization like `template<> `